libraries/core/src/main/java/com/paritytrading/philadelphia/FIXConnectionStatusListener.java

```diff
@@ -61,6 +61,15 @@ public interface FIXConnectionStatusListener {
 
     /**
      * Receive a Logon(A) message.
+     * <p>Please note that the incoming Logon(A) message might have a MsgSeqNum(34)
+     * value that is higher than the next expected MsgSeqNum(34) value.
+     * In this case, the application should immediately follow the Logon(A)
+     * response with a ResendRequest(2) message.
+     * <p>Also the standard suggests:<br>
+     * The initiator and acceptor should wait a short period of time following
+     * receipt of the Logon(35=A) message from the counterparty before
+     * transmitting queued or new application messages to permit both sides
+     * to synchronize the FIX session.
      *
      * @param connection the connection
      * @param message the Logon(A) message
```

Comment:
Let's add a reference to the new `FIXConnection#sendResendRequest()` method to this Javadoc after the `@throws` tag:
```java
/**
 * ...
 * @throws IOException if an I/O error occurs
 * @see FIXConnection#resendRequest
 */
```